frappe/public/js/frappe/ui/toolbar/subscription.js

```diff
@@ -0,0 +1,77 @@
+$(document).on("startup", async () => {
+	let response = await frappe.call({
+		method: "frappe.utils.subscription.show_banner",
+		callback: (response) => response,
```

Comment:
This callback doesn't do anything? 

Use `frappe.xcall` if using with await like this.